docs/annotated_mnist.md

```diff
@@ -8,7 +8,7 @@ import flax
 Load vanilla NumPy for use on host.
 
 ```py
-import numpy as onp
+import numpy as np
```

Comment:
Do we actually use this anywhere in the example? Otherwise let's simply remove the entire paragraph.

Reply:
Good point @andsteing

If NumPy isn't used, I'll make sure I remove the import